### ISSUE TITLE
Change in the aggregate file format

### ISSUE THREAD

**Issue by SimonaGh** [harrison-h/23andMe-Common-Relatives-Scraper#2]
The format of the link with the hash from the aggregate data file has changed in the meanwhile, so now the hash starts from character 32 rather than 59.

https://github.com/harrison-h/23andMe-Common-Relatives-Scraper/blob/c6d50b8bf5e632db866e400d617800ab3c8a62b9/23andMe%20Common%20Relatives%20Scraper.py#L45-L46

**Comment by kapsolas** [comment]
I think the issue is greater than just the ids. 
These can be parsed. I've updated this section of the code, but it now fails on the base URL.

Can someone confirm if this base URL still exists or if it has been changed to something else?

**Comment by SimonaGh** [comment]
The base URL has changed.
Instead of  " base_url = r'https://you.23andme.com/url/tools:compare_match/?remote_id={}' " ,it should be " base_url = r'https://you.23andme.com/profile/{}' "

**Comment by kapsolas** [comment]
Actually the base_url still works.

I had to use a different mechanism to get the relatives downloaded with the
remote ids and then run the process as before.

It looks like the original release of the script used a downloaded text
file for the remote_id. This no longer works. Instead if you poke around
the HTTP requests a bit, you can identify a request that returns the
information required in a JSON file, parse that file to populate the
relatives_list arrays and let the rest of the original process execute.

I haven't coded mine yet, but I will soon, i have all the required
information now.

**Comment by SimonaGh** [comment]
Yes, I was not able to scrape the "relative in common" data using the remote_id from the downloaded aggregate data.

Do you mind sharing that code once you do it?

**Comment by kapsolas** [comment]
I think this piece of the code is done. I’ll have to validate the results. If it all checks out, I’ll consider possibly releasing it.

**Comment by SimonaGh** [comment]
Let us know how you get on with that.
